_posts/2015-11-24-release-1.2.0.markdown

```diff
@@ -5,7 +5,9 @@ date:   2015-11-24
 ---
 
 We are pleased to announce the release of OMERO.figure 1.2.0.
-This release introduces support for ROIs on figure panels.
+This release introduces support for lightweight ROIs on figure panels.
```

Comment:
I wonder if 'illustrative ROIs' would be a better term - it is actually literally true as opposed to co-opting a term which doesn't have a clear meaning in this context.